cmd/startup/startup.go

```diff
@@ -386,15 +386,10 @@ func startIngestServer(serverAddr string) {
 		}
 	}()
 
-	err := metrics.RecoverWALData()
-	if err != nil {
-		log.Errorf("startIngestServer: Failed to recover WAL files, err: %v", err)
-	}
-	err = metrics.RecoverMNameWALData()
-	if err != nil {
-		log.Errorf("startIngestServer: Failed to recover Metrics Name WAL files, err: %v", err)
-		return
-	}
+	_ = metrics.RecoverWALData()
```

Comment:
you should differentiate the error between walk files not being present and wal recovery failing. 
They could be two different things. 
we should still log an error if we cannot recover a wal file but not log it as an error if no wal was present at all